daemon/src/projection.rs

```diff
@@ -1263,46 +1240,12 @@ impl From<xtra_bitmex_price_feed::Quote> for Quote {
     }
 }
 
-/// Maker offers represents the offers as created by the maker
 #[derive(Debug, Default, Clone, PartialEq, Serialize)]
 pub struct MakerOffers {
-    /// The offer where the maker's position is long
-    pub long: Option<CfdOffer>,
-    /// The offer where the maker's position is short
-    pub short: Option<CfdOffer>,
-}
-
-impl From<Option<model::MakerOffers>> for MakerOffers {
-    fn from(offers: Option<model::MakerOffers>) -> Self {
-        let (long, short) = match offers {
-            None => (None, None),
-            Some(offers) => {
-                let projection_long =
-                    offers
-                        .long
-                        .and_then(|long| match TryInto::<CfdOffer>::try_into(long) {
-                            Ok(projection_long) => Some(projection_long),
-                            Err(e) => {
-                                tracing::warn!("Unable to convert long offer: {e:#}");
-                                None
-                            }
-                        });
-
-                let projection_short =
-                    offers
-                        .short
-                        .and_then(|short| match TryInto::<CfdOffer>::try_into(short) {
-                            Ok(projection_short) => Some(projection_short),
-                            Err(e) => {
-                                tracing::warn!("Unable to convert short offer: {e:#}");
-                                None
-                            }
-                        });
-                (projection_long, projection_short)
-            }
-        };
-        MakerOffers { long, short }
-    }
+    pub btcusd_long: Option<CfdOffer>,
```

Comment:
🤔 This is a bit annoying, but on the `projection` this is acceptable to flatten out instead of using e.g. a `HashMap` as representation.

Reply:
It's not what I wanted to do originally, but it is the simplest thing I could come up with for the time being. Luckily we can change and break this as much as we want, even after releasing.